### PR TITLE
Support for prem_1b_chat

### PR DESCRIPTION
**Pull request by Killthebug**
update models.json file for prem_1b_chat support

### CODE REVIEW

File: prem_utils/models.json [hunk 1]
```diff
@@ -1339,9 +1339,9 @@
                     ]
                 },
                 {
-                    "slug": "prem-1b-chat",
+                    "slug": "prem_1b_chat",
                     "model_type": "text2text",
-                    "coming_soon": true,
```
Comment (nsosio) [inline prem_utils/models.json:1344]: why have we removed the coming_soon?

Comment (Killthebug) [reply prem_utils/models.json:1344]: Once released the coming_soon param becomes irrelevant. Followed the pattern of the other model definitions to just delete it and not make it False.

Do we wish to make this as `coming_soon: False`? That kinda seems redundant right?

Comment (nsosio) [reply prem_utils/models.json:1344]: @vittoriop17 @giowe do we need the coming_soon?

Comment (vittoriop17) [reply prem_utils/models.json:1344]: TLDR; I think it is correct to remove it if we wanna serve the model

@nsosio I think the flag is currently used to filter out these models and not show them in the platform. In one of the previous releases it was used to disabile the usage of the specified model (that was shown in the platform). But for clearness, I would wait @giowe's last word

**Comment by nsosio** [comment]
@Killthebug can you fix the conflicts?

**Comment by Killthebug** [comment]
tested with new prem-ml deployment and things are working good!

**Review by nsosio — APPROVED** [review]

**Pull request merged**